### ISSUE TITLE
[Accessibility]A11y_VSCode_Editor_Keyboard: On tab keypress, the first item of suggestion automatically gets selected.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52444]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated: MAS2.4.3

**Repro Steps:**
1. Launch VS Code.
2.Open any file in editor and type something.

**Actual:**
On tab keypress, the first item of suggestion automatically gets selected.

**Expected:**
First list item should not select on tab keypress or if this is happening, screenreader should announce some information or instruction about it.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

Attachment for Reference: 
[A11y VSCode Editor Jaws first element tab key.pptx](https://github.com/Microsoft/vscode/files/2118901/A11y.VSCode.Editor.Jaws.first.element.tab.key.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by jrieken** [comment]
@ramya-rao-a @joaomoreno doesn't announce the widget what's currently selected?

**Comment by auchenberg** [comment]
Closing this issue as it's only observed in JAWS, and this isn't strictly needed to meet MAS C compliance.
